server/model/agent.go

```diff
@@ -14,19 +14,26 @@
 
 package model
 
+import (
+	"errors"
+	"fmt"
+)
+
 type Agent struct {
 	ID          int64  `json:"id"            xorm:"pk autoincr 'id'"`
 	Created     int64  `json:"created"       xorm:"created"`
 	Updated     int64  `json:"updated"       xorm:"updated"`
 	Name        string `json:"name"          xorm:"name"`
-	OwnerID     int64  `json:"owner_id"      xorm:"'owner_id'"`
+	OrgID       int64  `json:"owner_id"      xorm:"'owner_id'"` // TODO: rename to org_id
 	Token       string `json:"token"         xorm:"token"`
 	LastContact int64  `json:"last_contact"  xorm:"last_contact"`
 	Platform    string `json:"platform"      xorm:"VARCHAR(100) 'platform'"`
 	Backend     string `json:"backend"       xorm:"VARCHAR(100) 'backend'"`
 	Capacity    int32  `json:"capacity"      xorm:"capacity"`
 	Version     string `json:"version"       xorm:"'version'"`
 	NoSchedule  bool   `json:"no_schedule"   xorm:"no_schedule"`
+	// Server side enforced agent filters
+	Filters map[string]string `json:"filters" xorm:"'filters' json"`
```

Comment:
What kind of filters do you have in mind that should be set through the UI instead of the labels env variable config of the agent itself?

Reply:
anything u can think of and users find probably more use-cases ...